src/SprykerEco/Zed/PunchoutCatalogs/Communication/Form/PunchoutCatalogSetupRequestConnectionTypeForm.php

```diff
@@ -62,4 +63,24 @@ protected function addSetupSubForm(FormBuilderInterface $builder)
 
         return $this;
     }
+
+    /**
+     * @param \Symfony\Component\Form\FormBuilderInterface $builder
+     *
+     * @return $this
+     */
+    protected function executeSetupRequestFormExtensionPlugins(FormBuilderInterface $builder)
+    {
+        $setupRequestFormExtensionPlugins = $this->getFactory()
+            ->getSetupRequestFormExtensionPlugins();
+
+        foreach ($setupRequestFormExtensionPlugins as $setupRequestFormExtensionPlugin) {
+            $builder->add(md5($setupRequestFormExtensionPlugin->getType()), $setupRequestFormExtensionPlugin->getType(), [
```

Comment:
why md5?

Reply:
Because I need some unique identifier, which is not changeable from request to request 
Discussed, agreed on md5